workflow_scripts/test_model_zoo.py

```diff
@@ -119,7 +119,7 @@ def main():
         print(
             f"In all {len(model_list)} models, {len(failed_models)} models failed, {len(skip_models)} models were skipped"
         )
-        for model, error in failed_messages:
+        for model, error in failed_messages:  # type: ignore
```

Comment:
Because it was not detected before, I assume a couple of annotations are inconsistent. It is a unit test so I did not look into it.

Reply:
```suggestion
        for model_name, error in failed_messages:
```
This should fix it. Thanks for catching.

Reply:
```suggestion
        for model_name, error in failed_messages:
            print(f"{model_name} failed because: {error}")
```
This hasn't been resolved yet?

Reply:
done